deps.sh

```diff
@@ -15,15 +15,20 @@ GOPATH_NEW="$(mktemp -d -t TEMP.XXXXXXX)"
 export GOPATH="${GOPATH_NEW}"
 
 get_deps() {
```

Comment:
should we have a comment explaining why we have to replace the apache thrift references with scoot thrift references (is it in case a vendor requires thrift?)